auth/src/main/java/com/firebase/ui/auth/ui/FlowParameters.java

```diff
@@ -94,9 +106,12 @@ public FlowParameters createFromParcel(Parcel in) {
             String termsOfServiceUrl = in.readString();
             int smartLockEnabledInt = in.readInt();
             int allowNewEmailAccountsInt = in.readInt();
+            int isReauthInt = in.readInt();
+            String reauthReason = in.readString();
 
             boolean smartLockEnabled = smartLockEnabledInt != 0;
             boolean allowNewEmailAccounts = allowNewEmailAccountsInt != 0;
+            boolean isReauth = isReauthInt != 0;
```

Comment:
@amandle The merge conflicts are my fault, sorry about that! I removed this unnecessary abstraction so this should work once you merge:
```java
boolean smartLockEnabled = in.readInt() != 0;
boolean allowNewEmailAccounts = in.readInt() != 0;
boolean isReauth = in.readInt() != 0;
String reauthReason = in.readString();
```